superset-frontend/src/dashboard/components/Header.jsx

```diff
@@ -86,6 +89,7 @@ const propTypes = {
   setMaxUndoHistoryExceeded: PropTypes.func.isRequired,
   maxUndoHistoryToast: PropTypes.func.isRequired,
   refreshFrequency: PropTypes.number.isRequired,
+  isPersistentRefreshFrequency: PropTypes.bool.isRequired,
```

Comment:
Nit: maybe name this `shouldPersistRefreshFrequency`, but up to you.

Reply:
I rename it to be `shouldPersistRefreshFrequency`